Lil-Peep/src/styles/modules/site-header.scss

```diff
@@ -44,34 +54,163 @@
         justify-content: flex-end;
         width: 50%;
       }
+    }
+  }
 
-      .customer-login {
-        li {
-          display: flex;
-          margin-right: 20px;
-        }
+  .customer-login {
+    li {
+      display: flex;
+      margin-right: 20px;
+    }
 
-        ul {
-          display: flex;
-          margin-right: 20px;
-        }
+    ul {
+      display: flex;
+    }
+  }
 
-        .site-logo {
-          display: flex;
+  .site-logo {
+    display: flex;
 
-          img {
-            display: block;
-            justify-content: center;
-            width: 100%;
-          }
+    img {
+      display: block;
+      justify-content: center;
+      width: 100%;
+    }
+  }
+
+  .navigation {
+    display: flex;
+
+    .nav-list {
+      padding-left: 10%;
+      width: 80%;
+    }
+
+    .cart-icon {
+      padding-left: 0;
+      width: 20%;
```

Comment:
Sometimes using percentage for widths is not recommendable because on large proportions the elements is going to increase more than expected, just make sure that doesn't happen in large screens.